fury/ui.py

```diff
@@ -60,6 +60,7 @@ def __init__(self, position=(0, 0)):
             UI component.
 
         """
+        self._scene = object()
```

Comment:
I use this dummy object to know when the callback should be added to the scene rather than an actor.

Reply:
See line 135 below for the setting the callback and line 4059 to see when I use `self._scene`.